R/duplicated.R

```diff
@@ -4,7 +4,7 @@ duplicated.data.table = function(x, incomparables=FALSE, fromLast=FALSE, by=seq_
     .NotYetUsed("incomparables != FALSE")
   }
   if (nrow(x) == 0L || ncol(x) == 0L) return(logical(0L)) # fix for bug #28
-  if (is.na(fromLast) || !is.logical(fromLast)) stopf("'fromLast' must be TRUE or FALSE")
+  if (is.na(fromLast) || !is.logical(fromLast)) stopf("'%s' must be TRUE or FALSE", "fromLast")
```

Comment:
Oh, didn't realize this one is an R message, sorry. Actually this won't lead to any consolidation since R- and C-level messages are in separate domains (.pot files). I don't see any similar enough messages at the R level, so let's revert this.